src/utils/autodownload.py

```diff
@@ -24,23 +25,26 @@ def autodownload(ref, save_dir, auth_token):
       paired_end is a boolean indicating if these are paired-end reads
     The generate_sketch function needs to know if it's working with paired-end reads or not
     """
-    ws_obj = ws.download_obj(ref=ref, auth_token=auth_token)
+    config = load_config()
+    ws = WorkspaceClient(url=config.get("workspace_url", "https://ci.kbase.us/services/ws"), token=auth_token)
```

Comment:
I think `config['workspace_url']` is guaranteed to be present in the dict, so no reason to use `.get` with a fallback I think